code/game/gamemodes/miniantags/slaughter/slaughter.dm

```diff
@@ -54,21 +58,35 @@
 
 /mob/living/simple_animal/slaughter/death()
 	..(1)
-	new /obj/effect/decal/cleanable/blood (get_turf(src))
-	var/obj/effect/decal/cleanable/blood/innards = new (get_turf(src))
-	innards.icon = 'icons/obj/surgery.dmi'
-	innards.icon_state = "innards"
-	innards.name = "pile of viscera"
-	innards.desc = "A repulsive pile of guts and gore."
-	new /obj/item/organ/internal/heart/demon (src.loc)
-	playsound(get_turf(src),'sound/magic/demon_dies.ogg', 200, 1)
-	visible_message("<span class='danger'>[src] screams in anger as it collapses into a puddle of viscera, its most recent meals spilling out of it.</span>")
-	for(var/mob/living/M in consumed_mobs)
-		M.loc = get_turf(src)
+	death_gibs()
```

Comment:
instead of this, put the stuff you want in the demon's loot var eg; `loot = list(/mob/living/simple_animal/pet/cat/kitten{name = "Laughter"}`
You don't need to set it on runtime either, you can set it as a var.